### PR TITLE
Generate docs from source: client

### PR DESCRIPTION
**Pull request by 0x6431346e**
I tried to merge what we had in:
- [docs](https://github.com/aragon/aragon.js/tree/master/docs)
- [hack:js-ref](https://github.com/aragon/hack/blob/master/docs/js-ref.md)
- source code

Note: only `APP.md` is generated from source, `PROVIDERS.md` and `WRAPPER.md` are not.

Potential improvements (will open new issues):
- [ ] generate wrapper docs from source
- [ ] Maybe do something about "Quick start" and "Background Scripts", they feel a bit redundant, especially since we are very descriptive in "APP.md". Feels like it should be a summary of this: https://hack.aragon.org/docs/tutorial.html#building-the-frontend

Part of #213

### CODE REVIEW

File: docs/APP.md [hunk 1]
```diff
@@ -1,49 +1,42 @@
-# Apps API Reference (`@aragon/client`)
+## Install
```
Comment (cjyabraham) [inline docs/APP.md]: Is this the right place for installation instructions? It seems unexpected to have these above the title of the page.

File: docs/WRAPPER.md [hunk 2]
```diff
@@ -1,6 +1,27 @@
-# Aragon Wrapper
+## Install
 
-## An Aragon wrapper.
+```sh
```
[the remark this answers is not in the document]
Comment (0x6431346e) [reply docs/WRAPPER.md]: Hmm, not sure, but I think we should have them somewhere, I can't see these instructions in the old docs at all.

**Comment by 0x6431346e** [comment]
> A few comments and a checkin. I'm still wanting to edit the source to APP.md if you can point me to where that is?

The source file of `APP.md` is in `packages/aragon-client/src/index.js`.
After editing the jsdoc there you need to run `npm run build:docs`

**Comment by 0x6431346e** [comment]
It looks like some updated dependencies bumped the wrapper size with `~90 KB` :disappointed:

File: docs/WRAPPER.md [hunk 3]
```diff
@@ -19,7 +21,6 @@ const AragonWrapper = require('@aragon/wrapper').default
 const providers = require('@aragon/wrapper').providers
 ```
 
-# API Reference
```
[the remark this answers is not in the document]
Comment (cjyabraham) [reply docs/WRAPPER.md]: Ok.  I didn't like how on hack.aragon you'd click on the menu item "Wrapper API Reference" and then not see that title at the top of the page.  Maybe we just need to use a different title for each page.  How about just "App", "Providers" and "Wrapper".  And then each page could have the different sections you suggested:   
- Install
- Import
- API Reference

**Pull request merged**